test/test_playbooks/auth_source_ldap.yml

```diff
@@ -0,0 +1,63 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - name: Load domain test-config
+    include_vars:
+      file: domain_vars.yml
+  - include_tasks: tasks/location.yml
+    vars:
+      location_name: "{{ item }}"
+      location_state: "present"
+    with_items: "{{ domain_locations }}"
+  - include_tasks: tasks/organization.yml
+    vars:
+      organization_name: "{{ item }}"
+      organization_state: "present"
+    with_items: "{{ domain_organizations }}"
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/auth_source_ldap.yml
+    vars:
+      auth_source_ldap_state: present
+      expected_change: true
+  - include: tasks/auth_source_ldap.yml
+    vars:
+      auth_source_ldap_state: present
+      expected_change: false
+  - include: tasks/auth_source_ldap.yml
+    vars:
+      auth_source_ldap_state: absent
```

Comment:
Can you add a test for updating an entity?

Reply:
Fixed